libs/core/src/lib/text/text.component.ts

```diff
@@ -0,0 +1,91 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    Input,
+    ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+    selector: 'fd-text',
+    templateUrl: './text.component.html',
+    styleUrls: ['./text.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TextComponent {
+    /**
+     * Text for render
+     */
+    @Input()
+    text: string;
+
+    /**
+     * Max visible lines of text
+     */
+    @Input()
+    maxLines = null;
+
+    /**
+     * Property allowing browsers to render whitespace and tabs
+     */
+    @Input()
+    whitespaces?: boolean;
+
+    /**
+     * Property for managing hyphenation, using css rule hyphens. Values: none | manual | auto
+     */
+    @Input()
+    hyphenation?: string;
```

Comment:
Should use a type here, improves the development experience for people using IDE's with smart completion

Reply:
Changed